tests/unit_tests/config/test_queue_config.py

```diff
@@ -127,3 +133,18 @@ def test_torque_queue_config_invalid_memory_pr_job(memory_with_unit_str):
         f.write(f"QUEUE_OPTION TORQUE MEMORY_PER_JOB {memory_with_unit_str}")
     with pytest.raises(ConfigValidationError):
         ErtConfig.from_file(filename)
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+def test_queue_option_LSF_SERVER_set_by_user_warning():
```

Comment:
The site wide fixture does not contain `QUEUE_OPTION`, but you could add a fixture instead that has some in `conftest.py`? The test is a lot better if we actually use site-config & user-config to test this.